### ISSUE TITLE
Large animals not dropping enough materials

### ISSUE THREAD

**Issue by Leon9621** [CleverRaven/Cataclysm-DDA#22433]
Game version:0.C-24983-g31e3fed 

Operating system: Windows 8

Tiles or curses: tiles

Mods active:None.

### Expected behavior

Butchering a moose / bear, both of which easily top 1,000 pounds as adults should yield an amount of meat / bones / sinew / organs / pelts somewhere near this total.

### Actual behavior

Butchering a moose / bear with an active circular saw, the best butchering tool, with 10 levels in survival yielded on average about 15-25 pounds in material, nowhere near as much as I believe should be provided. Even the in game stats reflect this; the body, before being butchered, of both of these animals weighs 264.55 pounds.

### Steps to reproduce the behavior

Set your survivor's survival skill to ten, find or spawn a circular saw with enough batteries to run and butcher a corpse, spawn or find and kill a moose / bear and butcher them with it.

**Comment by atealein** [comment]
It's same with fishing - you take out a salmon or tuna fish and it weights a lot, yet after butchering you get only few filet-o-fish.

**Comment by Mecares** [comment]
Game balance > realism

**Comment by atealein** [comment]
I believe the game balance lies in the fact that large animals are:
1) usually found in single numbers as opposed to flocks of smaller animals
2) far more dangerous to you and very hostile
3) you still will need high survival check to get the extra bits of meat out of them.

**Comment by ituluwituluwzev** [comment]
Large animal dropping realistic quantity of meat is not easy meat. Preserving meat requires tool, skill, and time, which early-game player character usually does not have. Those who happen to meet the requirement should be rewarded.

In winter start scenario, meat is significant food because cold temperature prevents rotting and plant-based food is not available. Siberian Yupik hunt one whale and eat the meat for half a year. Post-apocalyptic New Englander hunts a moose and lives on the meat for a week. It seems balanced and realistic.

**Comment by kevingranade** [comment]
> Game balance > realism

What's the balance being upset by this change?

**Comment by Coolthulhu** [comment]
> Preserving meat requires tool, skill, and time, which early-game player character usually does not have.

Depends how early game. Early-early, as in "too weak to kill a moose without kiting" or mid-early, as in "moose terror the murderhobo, reinventor of jerky" AKA day 4?
To make jerky, you only need a pan, a fire, a swamp, and cooking 3. Cooking 3 is the hardest part, but it's nothing you can't grind up before the end of the first week, you just need to commit to the grind.

That said, making current moose drop 200+kg of edible parts while still dying to hobo with a sharpened stick would be a very DDA thing. The food system is already broken by forest drops, so why not break it more with mounds of meat serving themselves up to be cut up with a piece of sharpened rock with a rag wrapped around it?
It's not like the survival in the wild is even the slightest bit challenging, it just requires getting spoiled on craft order.

**Comment by Treah** [comment]
@Coolthulhu 
lol its very true that wilderness is not hard at all for getting food... Water is actually more of a challenge if a swamp or river is not near by. That said tho even with a sharpend rock tied with a rag you would still get more then 3 or so strips of meat from a full sized moose. :P

**Comment by Barhandar** [comment]
>That said, making current moose drop 200+kg of edible parts while still dying to hobo with a sharpened stick would be a very DDA thing. 

Make large animals behave like actual animals (run away, attack only if threatened and can't run away, or has something to protect; absolutely gore the human unless the human is extremely good at dodging, have specialized weapons (like reach ones, IRL it's lugged spears) or can kill the large animal before it reaches them; add a very slowly depleting "fatigue" to animals only that is increased when they run and makes them collapse once it maxes out for persistence hunting, which is a danger in itself) and it won't be a problem.

Basically, if you want to focus on "nourishment management", make moose hard to kill (and large fish hard to catch).  If you don't, animals providing appropriate quantity of meat is not an issue.

**Comment by Coolthulhu** [comment]
If adequate changes happened before (or with) the huge buff to food produced, I wouldn't have an issue with it. It's just that so far, all "realism" changes have followed the same scheme of "features now, foundations and analysis necessary for them to work well much later (often never)".

> Basically, if you want to focus on "nourishment management", make moose hard to kill (and large fish hard to catch).

Still needs something to address the rotten jerky problem, which it greatly exacerbates. If the amount of processed food per corpse is supposed to be limited by time, the meat should be expected to rot mid-craft, changing the problem from "exploity bug that sometimes happens on its own" to "common bug expected to happen during typical use case".

There is also the issue of what constitutes being hard to kill and how to combine it with the existing system without it resulting in stupid random deaths from moose and bears charging the player on the other side of the map. Could require making them docile and harmless.

But knowing DDA development, all those issues will be ignored and we'll get hobos luring moose into swamps for easier processing, punching them in the faces, then turning their rotting corpses into mountains of fresh jerky (except for offal, which can't be preserved). All in the name of realism.

**Comment by Barhandar** [comment]
>Still needs something to address the rotten jerky problem, which it greatly exacerbates. 

Which would require significantly expanding on the crafting system to either block rotten ingredients from being made into non-rotting items (dumbest solution), increase amount of ingredients if they're rotten ("cutting away rotting parts") or equivalently being able to take some amount of "rotten" solid items and turn them into much smaller amount of non-rotten ones, or, most in line with classic roguelikes but clashing with the "MUH EASE OF PLAYING", make imperishables cooked with rotten ingredients _poisonous_.  
Note that the latter could also apply to perishables, after all, if you cook a soup you'll kill off the bacteria and so it's no longer "rotten", but it'll still have poisons produced by them (or worse, products of their thermal decay).

I'm not aware of the "rot mid-craft" issue, but logically food shouldn't be able to rot while it's being cooked. And actually pausing the timer leads to exploits with crafting cancels.
>It's just that so far, all "realism" changes have followed the same scheme of "features now, foundations and analysis necessary for them to work well much later (often never)".

Are you bitter about the broken ~~scurvy for everyone free despite real vitC being the most abundant one to the point you literally will die from mechanical damage to the digestive system sooner than from vitC poisoning~~ vitamin system, too?
>without it resulting in stupid random deaths from moose and bears charging the player on the other side of the map. Could require making them docile and harmless.

As I've said: they should run away by default and only attack when actively threatened (for CDDA purposes, if actually attacked by the player, damaged by a trap while player is in vicinity and they can see him, or when player comes too close to them - the latter needs some visuals as a "BACK OFF NOW" signal).
>There is also the issue of what constitutes being hard to kill

Lots of HP, lots of damage, hard to dodge due to size, charge attacks with knockback. Bonus if they run away if they take significant damage, as they do IRL.

**Comment by mathkr** [comment]
I think the amount of meat a moose drops is just **too** unrealistic right now to the point of feeling buggy/unpolished. Even if you account for game balance it is still a scratch-your-head moment when you butcher it and wonder if you did something wrong.

Maybe just multiplying the amount by ~5 could strike a balance?

**Comment by kevingranade** [comment]
As I commented before, I don't see a serious balance problem with the proposed change.  
A few notes though:  
https://www.pourvoiries.com/en-us/2012/02/how-many-pounds-of-meat-will-you-get-from-your-moose/ indicates that you might expect ~300lbs of meat from a ~1,000lb moose, IF it was a clean kill, field dressed properly, and butchered by a professional with a full set of tools.  
Several of these factors are either not tracked or impossible to achieve in-game, so that 300lbs can drop significantly, perhaps down to 200lbs of meat in the best case for the game, and degrading further from there for poor tools or skills, I wouldn't ever expect to see that drop under 100lbs of meat though, no matter how bad at butchering you are.
Regarding aggression, my understanding is that moose aren't particularly afraid of people, so you can aporoach if you like, they might attack if you get close, they are however more likely than not to leave if you get agressive.

Side note, the corpse weight needs to be bumped up to 1,000lbs if were using that as the starting weight.

So, tl;dr short term proposal:  
* Corpse weight = 1,000lbs
* worst meat yield = 100lbs
* best meat yield = 200lbs
* consider making moose slightly more dangerous and/or quicker to run away.

**Comment by Treah** [comment]
I dunno if you would ever want to change moose since they are the posterchild for CDDA deaths.

**Comment by codemime** [comment]
The problem is: meat is way too easy to obtain. You can get loads of it pretty much everywhere (forests, swamps, even cities). Every last insect drops it. Find a couple of spider basements, and you're all set for a very looong time. In short, I'd also propose this:
- Insects shouldn't be dropping meat at all (but something else instead)
- Reconsider https://github.com/CleverRaven/Cataclysm-DDA/pull/21968 which was rejected back then, but IMO looks like a good idea, especially in the light of this change
- Zombies hunting wildlife (arguably, but still)
